src/content/docs/en/guides/migrate-to-astro/from-create-react-app.mdx

```diff
@@ -339,7 +339,7 @@ import { getCollection } from 'astro:content';
 const allBlogPosts = await getCollection('blog');
 
 // Get all `src/pages/posts/` entries
-const allPosts = await Object.values(await import.meta.glob('../pages/post/*.md', { eager: true }));
+const allPosts = await Object.values(import.meta.glob('../pages/post/*.md', { eager: true }));
```

Comment:
Oh I noticed this one while translating, I think the `await` in front of `Object.values` is also useless here... This is not a Promise:
```suggestion
const allPosts = Object.values(import.meta.glob('../pages/post/*.md', { eager: true }));
```